### ISSUE TITLE
Postman Linux update removed all directories in the parent directory

### ISSUE THREAD

**Issue by onionka** [a85/POSTMan-Chrome-Extension#6031]
**Describe the bug**
When I ran a Postman today, it started to sync, and after a while, I noticed that it removed all files and directories where the Postman was stored.

**To Reproduce**
1. Run Postman
2. Wait a few seconds
3. Check directory when Postman was stored
4. See there are no Postman or any other programs I had there

**Expected behavior**
It seems like it should remove just Postman directory when it was updating itself

**App information (please complete the following information):**
 - App Type - Native App
 - Postman Version - don't know because it was removed
 - OS: Manjaro Illyria 18.0.3

**Comment by harryi3t** [comment]
@Cmiroslaf Any chance you remember how you updated the Postman app last time? Did you manually downloaded the `Postman-linux-x64-x.y.z.tar.gz` file and uncompressed it yourself or did the app updated itself (When this happens, you would see a toast message "Successfully updated to version ....")?

**Comment by onionka** [comment]
@harryi3t Probably a 4 months ago I have downloaded Postman native app and let it update by itself. I wasn't updating it manually at all.

I had program directory structure in home directory like this:

```
~/Applications
├── Postman
├── PyCharm
└── ...
```

Postman was downloading `Postman-linux-x64-x.y.z.tar.gz` packages into the `~/Applications` directory, so I had multiple versions of Postman unpacked there.

```
~/Applications
├── Postman
├── Postman-linux-x64-x.y.z
├── Postman-linux-x64-x.y.z
├── etc.
├── PyCharm
└── ...
```

I was always seeing the "Successfully updated to version ..." message after every update and was occationally removing the old versions.

**Comment by harryi3t** [comment]
@Cmiroslaf Thanks for the information. To help us further narrow the issue down, could you share the logs which you could find at `~/.config/Postman/logs`. You can share them at help@getpostman.com after redacting. Mention this issue number (#6031) in the subject of the mail.

**Comment by numaanashraf** [comment]
This issue has been addressed in Postman v7.0.7
